packages/dnb-eufemia/src/components/info-card/InfoCard.tsx

```diff
@@ -0,0 +1,240 @@
+import React from 'react'
+import classnames from 'classnames'
+
+// Components
+import Button from '../button/Button'
```

Comment:
```suggestion
import Button, { ButtonProps } from '../button/Button'
```

And then we can remove line 21

Reply:
Done